File: problems/3541_find_most_frequent_vowel_and_consonant/3541_find_most_frequent_vowel_and_consonant.java
class Solution {
    public int maxFreqSum(String s) {
        int maxVowel = 0;
        int maxConsonant = 0;
        int[] counts = new int[26];
        for (char c : s.toCharArray()) {
            int idx = c - 'a';
            counts[idx]++;
            if (isVowel(c)) {
                maxVowel = Math.max(maxVowel, counts[idx]);
            } else {
                maxConsonant = Math.max(maxConsonant, counts[idx]);
            }
        }
        return maxVowel + maxConsonant;
    }

    private boolean isVowel(char c) {
        return c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u';
    }
}

File: problems/3541_find_most_frequent_vowel_and_consonant/3541_find_most_frequent_vowel_and_consonant.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        cnt = Counter(s)
        max_vowel = 0
        max_consonant = 0
        for c, v in cnt.items():
            if c in "aeiou":
                max_vowel = max(max_vowel, v)
            else:
                max_consonant = max(max_consonant, v)
        return max_vowel + max_consonant